tracer/src/Datadog.Trace/AppSec/Coordinator/HttpTransportBase.cs

```diff
@@ -32,7 +34,13 @@ internal abstract class HttpTransportBase
     /// <summary>
     /// Disposes the WAF's context stored in HttpContext.Items[]. If it doesn't exist, nothing happens, no crash
     /// </summary>
-    internal void DisposeAdditiveContext() => GetAdditiveContext()?.Dispose();
+    internal void DisposeAdditiveContext()
+    {
+        GetAdditiveContext()?.Dispose();
+        _isAdditiveContextDisposed = true;
+    }
+
+    internal bool AdditiveContextDisposed() => _isAdditiveContextDisposed;
```

Comment:
nit: I find this a little confusing, a property `IsAdditiveContextDisposed` would better, I think?

Reply:
Done!